### PR TITLE
Implement a force option to proceed with depreciated schema.

### PR DESCRIPTION
**Pull request by davidewatson**

### CODE REVIEW

File: ansible/roles/kraken.config/defaults/main.yaml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 kraken_action: up
+config_forced: "{{ config_forced }}"
```
Comment (joejulian) [inline ansible/roles/kraken.config/defaults/main.yaml]: isn't this redundant?

**Comment by coffeepac** [comment]
retest this please

**Comment by coffeepac** [comment]
retest this please

**Comment by coffeepac** [comment]
pipelet passed, that's enough for a merge.  ONWARD

**Pull request merged**